### ISSUE TITLE
Suggestion: include note to add `assembleDebugAndroidTest` to Gradle Runner

### ISSUE THREAD

**Issue by mikemee** [bitrise-steplib/steps-virtual-device-testing-for-android#18]
**TL;DR:  Please add a note that `assembleDebugAndroidTest` needs to be added to Gradle Runner for this step to work.**

As a new user, I banged my head against this for a while.

Essentially my flow was: 

 * use all the default stuff after adding a pre-existing Android project
 * installed the `gradle-unit-test` step -> worked as expected (except I can't find the error logs when it fails)
 * installed the `virtual-device-testing-for-android` step, switched it to `instrumentation`

Then I kept getting an error: 

    Issue with TestApkPath: parameter not specified

After much head banging and many test runs, in digging around the archives I found the suggestion to add this to the Gradle Runner step:

    assembleDebugAndroidTest

So I did that, and magic happened!

I.e. my Gradle Runner config is now:

    - gradle-runner:
        inputs:
        - gradle_file: "$GRADLE_BUILD_FILE_PATH"
        - gradle_task: assembleDebug assembleDebugAndroidTest
        - gradlew_path: "$GRADLEW_PATH"

**Comment by doomsayer13** [comment]
Hi @mikemee,
Thanks for reporting an issue, we'll check it ASAP.

**Comment by trapacska** [comment]
Hey @mikemee 

The new version of the step now contains your suggestion, thanks!
https://github.com/bitrise-steplib/steps-virtual-device-testing-for-android/releases/tag/1.0.2

Happy Building! 🚀
